src/colors.tsx

```diff
@@ -0,0 +1,2 @@
+export const PRIMARY = '#1da57a';
```

Comment:
Same comment as less file, lets define the colors separately and assign values here to prevent confusion

Reply:
Im a bit confused by what you mean by this? Can you elaborate and suggest what I should change? I dont think its possible to share one common file that will sync across both the antd less styles and colors as JS values.